src/repository/storage/postgres/orm/sequelize/note.ts

```diff
@@ -187,6 +188,25 @@ export default class NoteSequelizeStorage {
     return note;
   }
 
+  /**
+   * Gets note list by creator id
+   *
+   * @param creatorId - note creator id
+   * @returns { Promise<NoteList | null> } note
+   */
+  public async getNoteListByCreatorId(creatorId: NoteCreatorId): Promise<NoteList | null> {
+    const noteList = await this.model.findAll({
+      where: {
+        creatorId,
+      },
+    });
+
+    if (noteList.length === 0) {
```

Comment:
I think, this check is redundant here

Reply:
Left a check for an empty array so that null is returned and not item[]

Reply:
Remove